public/app/plugins/panel/dashlist/panelcfg.gen.ts

```diff
@@ -12,14 +12,13 @@ export const PanelCfgModelVersion = Object.freeze([0, 0]);
 
 export enum PanelLayout {
   List = 'list',
-  Previews = 'previews',
 }
 
 export interface PanelOptions {
   folderId?: number;
   includeVars: boolean;
   keepTime: boolean;
-  layout?: PanelLayout;
+  layout?: 'list';
```

Comment:
hmm.. I wonder if this is intended - `@cuetsy(kind="enum")` is still there, so shouldn't this be an enum?